Modules/_interpretersmodule.c

```diff
@@ -0,0 +1,1892 @@
+
+/* interpreters module */
+/* low-level access to interpreter primitives */
+
+#include "Python.h"
+#include "frameobject.h"
+#include "internal/pystate.h"
+
+
+static PyInterpreterState *
+_get_current(void)
+{
+    PyThreadState *tstate;
+
+    tstate = PyThreadState_Get();
+    if (tstate == NULL)
```

Comment:
You don't need to check for NULL here, `PyThreadState_Get` aborts if it can't lookup the state.

Reply:
fixed